data_preprocessing/main.py

```diff
@@ -4,6 +4,7 @@
 def main():
     columns_to_filter = ["TotPkts", "TotBytes", "SrcBytes", "flag", "service", "count", "dst_bytes", "class"]
     columns_to_transform_to_log = ["TotPkts", "SrcBytes", "dst_bytes"]
+    columns_to_encode = ["Flag", "Service", "Class"]
```

Comment:
The encode part is before changing names to PascalCase, so when this part is executed it has old column names. It has to be switched to old names or moved encoding part after `data_processor.change_column_names_to_pascal_case()` execution

Reply:
I've changed the column names to the old ones